packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -1060,4 +1065,33 @@ class TextPainter {
     assert(_debugAssertTextLayoutIsValid);
     return  _lineMetricsCache ??= _paragraph!.computeLineMetrics();
   }
+
+  bool _disposed = false;
+
+  /// Whether this object has been disposed or not.
+  ///
+  /// Only for use when asserts are enabled.
```

Comment:
I think we typically refer to it as "debug mode", since on the command-line the option is `--debug`.

Reply:
Some customers disable asserts in debug mode.